### PR TITLE
Integrate Test coverage (nyc)

### LINKED ISSUE #104: Display test coverage
We need to know test coverage in all packages.
I think the best tool is [istanbuljs/nyc](https://github.com/istanbuljs/nyc).

### PR DESCRIPTION
**Pull request by nd-02110114**
Fixed #104 
Fixed #106

### CODE REVIEW

File: modules/is-restorable/.nycrc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (Leko) [inline modules/is-restorable/.nycrc]: I think `.nycrc` should be placed in root directory.
And replace package.json test script with `--nycrc-path` option.

**Comment by nd-02110114** [comment]
@Leko 
Thank you for reviewing.
I modified by following your advice. Please confirm 🙇

**Comment by shinout** [comment]
@nd-02110114 Sorry, could you rebase this PR to master?

**Comment by nd-02110114** [comment]
@shinout 
I rebased this PR.

**Comment by shinout** [comment]
Thanks!

**Pull request merged**